src/code/z_parameter.c

```diff
@@ -4937,64 +4980,1081 @@ void Interface_DrawMinigameIcons(PlayState* play) {
+    }
+
+    if (BUTTON_ITEM_EQUIP(0, EQUIP_SLOT_C_RIGHT) < 0xF0) {
+        Interface_LoadItemIconImpl(play, EQUIP_SLOT_C_RIGHT);
+    }
+
+    if (((gSaveContext.timerStates[TIMER_ID_MINIGAME_2] == TIMER_STATE_COUNTING) ||
+         (gSaveContext.timerStates[TIMER_ID_GORON_RACE_UNUSED] == TIMER_STATE_COUNTING)) &&
+        ((gSaveContext.respawnFlag == -1) || (gSaveContext.respawnFlag == 1)) &&
+        (gSaveContext.timerStates[TIMER_ID_MINIGAME_2] == TIMER_STATE_COUNTING)) {
+        gSaveContext.timerStates[TIMER_ID_MINIGAME_2] = TIMER_STATE_START;
+        gSaveContext.timerX[TIMER_ID_MINIGAME_2] = 115;
+        gSaveContext.timerY[TIMER_ID_MINIGAME_2] = 80;
+    }
+
+    LifeMeter_Init(play);
+    Map_Init(play);
+
+    gSaveContext.minigameStatus = MINIGAME_STATUS_END;
+    interfaceCtx->perfectLettersPrimColor[0] = 255;
+    interfaceCtx->perfectLettersPrimColor[1] = 165;
+    interfaceCtx->perfectLettersPrimColor[2] = 55;
+
+    if (CHECK_EVENTINF(EVENTINF_34)) {
+        CLEAR_EVENTINF(EVENTINF_34);
+        gSaveContext.timerStates[TIMER_ID_MINIGAME_2] = TIMER_STATE_OFF;
+    }
+
+    gSaveContext.timerStates[TIMER_ID_MINIGAME_1] = TIMER_STATE_OFF;
+    gSaveContext.timerCurTimes[TIMER_ID_MINIGAME_1] = SECONDS_TO_TIMER(0);
+    gSaveContext.timerTimeLimits[TIMER_ID_MINIGAME_1] = SECONDS_TO_TIMER(0);
+    gSaveContext.timerStartOsTimes[TIMER_ID_MINIGAME_1] = 0;
+    gSaveContext.timerStopTimes[TIMER_ID_MINIGAME_1] = 0;
+    gSaveContext.timerPausedOsTimes[TIMER_ID_MINIGAME_1] = 0;
+
+    for (i = 0; i < TIMER_ID_MAX; i++) {
+        if (gSaveContext.timerStates[i] == TIMER_STATE_7) {
+            gSaveContext.timerStates[i] = TIMER_STATE_OFF;
+        }
+    }
+
+    sPictoState = PICTO_BOX_STATE_OFF;
+    sPictoPhotoBeingTaken = false;
+
+    if ((play->sceneId != SCENE_MITURIN_BS) && (play->sceneId != SCENE_HAKUGIN_BS) && (play->sceneId != SCENE_SEA_BS) &&
+        (play->sceneId != SCENE_INISIE_BS) && (play->sceneId != SCENE_LAST_BS) && (play->sceneId != SCENE_MITURIN) &&
+        (play->sceneId != SCENE_HAKUGIN) && (play->sceneId != SCENE_SEA) && (play->sceneId != SCENE_INISIE_N) &&
+        (play->sceneId != SCENE_INISIE_R) && (play->sceneId != SCENE_LAST_DEKU) &&
+        (play->sceneId != SCENE_LAST_GORON) && (play->sceneId != SCENE_LAST_ZORA) &&
+        (play->sceneId != SCENE_LAST_LINK)) {
+
+        CLEAR_EVENTINF(EVENTINF_53); // Goht intro cutscene watched
+        CLEAR_EVENTINF(EVENTINF_54); // Odolwa intro cutscene watched
+        CLEAR_EVENTINF(EVENTINF_55); // Twinmold intro cutscene watched
+        CLEAR_EVENTINF(EVENTINF_56); // Gyorg intro cutscene watched
+        CLEAR_EVENTINF(EVENTINF_57); // Igos du Ikana intro cutscene watched
+        CLEAR_EVENTINF(EVENTINF_60); // Wart intro cutscene watched
+        CLEAR_EVENTINF(EVENTINF_61); // Majoras intro cutscene watched
+        CLEAR_EVENTINF(EVENTINF_62); //
+        CLEAR_EVENTINF(EVENTINF_63); // Gomess intro cutscene watched
```

Comment:
Any reason to not name those eventinf yet?

Reply:
This is based off old knowledge, and while it seems to hold generally true from my testing, it's unclear if it's more specific than this. So it's best to name these along with the respective bosses imo

Reply:
I just checked the decomped (and merged) ones:
- Gyorg seems to use  this eventinf for checking if the cutscene has been watched already
- Twinmold is hard to say without a deeper dive into the code
- I couldn't find where Igos du Ikana sets EVENTINF_57, but the curtains check its value
- Wart seems to use this eventinf for the cutscene too

So I think you could at least name `EVENTINF_56` and `EVENTINF_60`, but it is OK to me if you prefer not to name them yet

Reply:
Think my preference is to hold on, I don't understand bosses code well enough to be confident in exactly the use of these flags